source/priority_queue.c

```diff
@@ -145,11 +145,7 @@ int aws_priority_queue_init_dynamic(
     queue->pred = pred;
     AWS_ZERO_STRUCT(queue->backpointers);
 
-    
-    bool ret = aws_array_list_init_dynamic(&queue->container,
-                                           alloc,
-                                           default_size,
-                                           item_size);
+    bool ret = aws_array_list_init_dynamic(&queue->container, alloc, default_size, item_size);
```

Comment:
Formatting change?

Reply:
I think that clang-format did that change.